packages/@okta/vuepress-site/guides/sign-into-your-web-app/require-authentication/netcore/reqautheverything.md

```diff
@@ -1 +1,26 @@
-code example
\ No newline at end of file
+To require authentication for all actions you can create an authorization policy to be used everywhere:
+
+```
+public void ConfigureServices(IServiceCollection services)
+{
+    services.AddMvc(o =>
+    {
+        var policy = new AuthorizationPolicyBuilder()
+            .RequireAuthenticatedUser()
+            .Build();
+        o.Filters.Add(new AuthorizeFilter(policy));
+    });
+}
+```
+
+Also, you can opt to create a `BaseProtectedController` decorated with the `Authorize` attribute and make all your controllers to inherits it:
```

Comment:
`and make all your controllers inherit it:`

Reply:
Done